doc/whats_new.rst

```diff
@@ -104,6 +104,23 @@ Bug fixes
      the same result as the LassoLars implementation available
      in R (lars library). :issue:`7849` by `Jair Montoya Martinez`_
 
+   - Fixed a bug where :func:`sklearn.model_selection.train_test_split` raised
```

Comment:
those should be under 0.18.1.
Also, some of the links with your username don't render properly in the 0.18.1 whatsnew. Would you mind checking?